comps/comps-pulpcore-el7.xml

```diff
@@ -163,6 +163,12 @@
       <packagereq type="default">rh-postgresql12-postgresql-debversion</packagereq>
       <packagereq type="default">rh-postgresql12-postgresql-evr</packagereq>
       <packagereq type="default">ruby-solv</packagereq>
+      <packagereq type="default">tfm-python38</packagereq>
```

Comment:
I was curious why name it a python SCL vs. a Pulp SCL?

Reply:
I don't mind either way.
My trail of thought was: we're providing Python to run Pulp in, so Python.

Reply:
Will the Pulp packages also be in this SCL?

Reply:
Yeah

Reply:
So then I think I owuld lean towards either:

```
tfm-pulpcore
pulpcore-python38
```

I think without the pulp name in there, folks looking at the packages and debugging via `scl enable` will get confused and we'll field more questions than we want.

Reply:
I kinda dislike the `pulpcore` prefix, as this (for me) implies that the SCL is somehow related to the Pulp *Project*, which it isn't (and shouldn't be).

Onother thing that came to my mind when you asked about the prefix yesterday: what will/should happen if we move to Python 3.10 (or whatever)? For Foreman, we've been keeping the `tfm` SCL while switching Rubies and Nodes underneath. And while this is nice for the "you can just install `tfm-<whatever>` without thinking which Ruby version it uses", it always struck me a bit odd as verifying a change in the underlying stack always required looking at the actual dependencies of the built RPMs. But then again it's really nice for the installer and friends…

Voting `tfm-pulpcore` then.

What about you?

Reply:
It also means you can `yum update` and upgrade. If we'd introduce a version in the SCL name then that becomes much more complicated. I'm not sure the packaging templates have `Obsoletes` and obsoleting something from another SCL feels weird.

:+1: for `tfm-pulpcore`.